experimental/packages/opentelemetry-sdk-metrics-base/src/export/MetricData.ts

```diff
@@ -87,9 +93,6 @@ export interface CollectionResult {
  * The aggregated point data type.
  */
 export enum DataPointType {
-  /**
-   * A singular metric data point has only a single numeric value.
-   */
   SINGULAR,
```

Comment:
Do we still need singular?

Reply:
No, we do not. Meant to remove this too. Thanks for catching this :slightly_smiling_face: 
Removed in [d9de8d3](https://github.com/open-telemetry/opentelemetry-js/pull/3079/commits/d9de8d3a19581ba782f468ff9290a4c662d00ab1).